LayoutTests/fullscreen/full-screen-keyboard-lock-option-enabled.html

```diff
@@ -0,0 +1,57 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <title>This tests that full screen mode with keyboard lock option allows ESC key event.</title>
+</head>
+<body>
+    <div style="container-type: size; width: 100%; height: 100%;" id="target">
```

Comment:
Is there a reason to use `container-type: size` here?

Reply:
@nt1m  Thanks, fixed it.